Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx

```diff
@@ -580,8 +578,8 @@ ConstNeighborhoodIterator<TImage, TBoundaryCondition>::PrintSelf(std::ostream &
   {
     os << m_Region.GetSize()[i] << " ";
   }
-  os << "} }";
-  os << ", m_BeginIndex = { ";
+  os << "} }"
+     << ", m_BeginIndex = { ";
```

Comment:
Why didn't this fit onto the previous line?

Reply:
Good question, but that's up to clang-tidy! Even when I remove the line-break, clang-tidy puts it back, in this case!

A follow-up PR could merge it together to `  os << "} }, m_BeginIndex = { "`, of course.  Clang-tidy would allow that on one single line.